src/Migrations/Version20200916093101.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20200916093101 extends AbstractMigration
```

Comment:
Shoudn't it be loaded automatically from the `CoreBundle`?

Reply:
It is opened to 1.7, where we don't have new migrations yet. During upmerge it should be removed

Reply:
Indeed 👍